spring-cloud-stream/src/main/java/org/springframework/cloud/stream/endpoint/ChannelsEndpoint.java

```diff
@@ -56,10 +56,12 @@ public ChannelsEndpoint(List<Bindable> adapters,
 			for (String name : factory.getInputs()) {
 				inputs.put(name, bindings.containsKey(name) ? bindings.get(name)
 						: new BindingProperties());
+				// TODO - add consumer properties
```

Comment:
would be a good idea to create a GH issue for this TODO if it won't be address in the scope of this PR

...same for the one a few lines below